src/redturtle/prenotazioni/browser/utilities/notify_upcoming_bookings.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from datetime import timedelta
+from logging import getLogger
+
+from plone import api
+from Products.Five.browser import BrowserView
+from zope.event import notify
+
+from redturtle.prenotazioni.events import BookingReminderEvent
+
+logger = getLogger(__name__)
+
+
+class NotifyUpcomingBookings(BrowserView):
+    """View to notify about the upcomming bookings"""
+
+    def __call__(self):
+        today = datetime.now().replace(hour=0, minute=0)
+        unrestricted_search = api.portal.get_tool(
+            "portal_catalog"
+        ).unrestrictedSearchResults
+
+        for brain in unrestricted_search(portal_type="PrenotazioniFolder"):
```

Comment:
i would not use unrestricted search here.
If you want to get all the items, then call this view as a manager.